### ISSUE TITLE
Css Customization & Support

### ISSUE THREAD

**Issue by Shailth** [andyyou/react-coverflow#57]
Hi Folks,

I would say tremendous effort to bring up such a fine and precise work.
For sure a re-usable and easy to implement component.
Would request the team to provide help on the following enhancements:
1) The parameter/prop list supplied to CoverFlow to hold custom updates to css like background color/fonts/etc.
2) Have navigation buttons on the sides rather than bottom(other related customization options if possible.)

Thanks a ton,
Shalabh

**Comment by Shailth** [comment]
Hi Asalem,
i have done some work in this direction, would like to contribute if you allow.
Also, i have some ideas further to enrich CoverFlow, if you please to discuss, at your convenience.

<img width="1156" alt="screen shot 2019-01-24 at 12 34 57 pm" src="https://user-images.githubusercontent.com/13822096/51678520-81a34700-1fd4-11e9-8bee-05823900c511.png">

**Comment by Asalem1** [comment]
@Shailth that sounds great! I'm definitely open to checking out PRs and working together to implement some of the changes you're looking to make

**Comment by nojaf** [comment]
Hello, any update on this? I would like to customize the background of the container and the colours of the alt text. Pretty clueless about how to proceed here.

**Comment by Asalem1** [comment]
Hi @nojaf , I still haven't heard back from @Shailth about implementing this feature. If you'd like to send a PR and contribute to the project you're more than welcome to make additions

**Comment by nojaf** [comment]
Hi @Asalem1 , I currently have a version where I skip the entire Radium thing. Modifying the scss file directly better fitted my workflow. Would that be an option as well?

**Comment by Asalem1** [comment]
@nojaf , definitely. I mentioned in #53 that I'm looking to remove the project's dependency on Radium so that would be great!
